### PR TITLE
Use cloudflare lib, fixes #4.

### LINKED ISSUE #4: Helm chart
See https://github.com/mozmar/cloudflare-datadog/pull/3#issue-213566077

### PR DESCRIPTION
**Pull request by glogiotatidis**
@jgmize r?

/cc @so0k

### CODE REVIEW

**Comment by so0k** [comment]
Need to add `config.py` to [COPY instruction](https://github.com/mozmar/cloudflare-datadog/blob/9ab840a1f352c9df9744290c2d7a439b19847e10/Dockerfile#L9) in Dockerfile?

a personal nit - I know it's a breaking change, but ... can't we just use the default env names for cloudflare lib? 

```
CF_API_EMAIL
CF_API_KEY
```

then don't need for `config.py` to handle those 2 vars, just let the cloudflare library handle it.

**Comment by so0k** [comment]
also, the indentation on the README broke (the notes on the `SINCE` argument should be in a sub list).

We could use a markdown table for the arguments (similar to the Helm doc conventions)

**Comment by glogiotatidis** [comment]
> then don't need for config.py to handle those 2 vars, just let the cloudflare library handle it.

We'll still need those in `config.py` because the cf lib will only load from env variables and not from `.env`. It's also nicer to have everything explicitly defined

**Comment by glogiotatidis** [comment]
> a personal nit - I know it's a breaking change, but ... can't we just use the default env names for cloudflare lib?

Indeed a breaking change but indeed it makes sense b/c current naming is not ideal.

FTR I already updated our deployment with the new variables in addition to the current ones, so when this gets merged it just works.

File: README.md [hunk 1]
```diff
@@ -1,87 +1,84 @@
...
 
-Populate `AUTH_EMAIL` and `AUTH_KEY` parameters, then:
 ```
 make build
 make shell
-apk add --no-cache jq curl
-curl https://api.cloudflare.com/client/v4/zones?name=example.com \
- -H "X-Auth-Email: $AUTH_EMAIL" -H "X-Auth-Key: $AUTH_KEY" \
- | jq -r .result[].id
+./getZoneIDs.py
```
Comment (bookshelfdave) [inline README.md:59]: btw, the Cloudflare UI shows zoneid on the front page

**Comment by bookshelfdave** [comment]
> FTR I already updated our deployment with the new variables in addition to the current ones, so when this gets merged it just works.

Do you mean the k8s deployment I installed yesterday?

**Review by bookshelfdave — COMMENTED** [review]
:+1:

File: README.md (hunk 1 above)
Comment (so0k) [reply README.md:59]: yeah, I noticed that yesterday :o

**Comment by so0k** [comment]
should these instructions on getting timezone information be added to the README as well?

https://github.com/mozmar/cloudflare-datadog/pull/3#issuecomment-286487180

**Comment by glogiotatidis** [comment]
> Do you mean the k8s deployment I installed yesterday?

no, the one running on deis_us @ cloudflare-datadog-sync

**Review by jgmize — APPROVED** [review]

**Comment by glogiotatidis** [comment]
Thanks all!

**Pull request merged**